src/m365/teams/commands/chat/chat-get.ts

```diff
@@ -0,0 +1,200 @@
+import { AadUserConversationMember, Chat, ConversationMember } from '@microsoft/microsoft-graph-types';
+import { AxiosRequestConfig } from 'axios';
+import Auth from '../../../../Auth';
+import * as os from 'os';
+import request from '../../../../request';
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import GraphCommand from '../../../base/GraphCommand';
+import commands from '../../commands';
+import { validation } from '../../../../utils/validation';
+import { accessToken } from '../../../../utils/accessToken';
+import { ODataResponse } from '../../../../utils/odata';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id?: string;
+  participants?: string;
+  name?: string;
+}
+
+class TeamsChatGetCommand extends GraphCommand {
+  public get name(): string {
+    return commands.CHAT_GET;
+  }
+
+  public get description(): string {
+    return 'Get a chat conversations by id, participants or chat name';
+  }
+
```

Comment:
Since we have some optional options in this command, let's add tracking their usage to our telemetry